jetty-util/src/main/java/org/eclipse/jetty/util/component/DumpableContainer.java

```diff
@@ -0,0 +1,36 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2020 Mort Bay Consulting Pty Ltd and others.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.util.component;
+
+/**
+ * DumpableContainer
+ * 
+ * A Dumpable that is a container of beans can implement this
+ * interface to allow it to refine which of its beans can be
+ * 
+ */
+public interface  DumpableContainer extends Dumpable
```

Comment:
Double space after `interface`!

Reply:
I would move this simple class as an inner class of `Dumpable`, namely `Dumpable.Container`.

Reply:
It can't be called Dumpable.Container because it conflicts in many classes which import both org.eclipse.jetty.util.component.Container and org.eclipse.jetty.util.component.Dumpable, so there would be a lot of places where currently parameters, variables and fields would have to be fully qualified to org.eclipse.jetty.util.component.Container. So I settled on Dumpable.DumpableContainer.